cmd/admin-user-add.go

```diff
@@ -115,6 +123,35 @@ func (u userMessage) JSON() string {
 	return string(jsonMessageBytes)
 }
 
+// fetchUserKeys - returns the access and secret key
+func fetchUserKeys(args cli.Args) (string, string) {
+	accessKey := ""
+	secretKey := ""
+	console.SetColor(cred, color.New(color.FgYellow, color.Italic))
+	reader := bufio.NewReader(os.Stdin)
+
+	switch numberOfArgs := len(args); {
+	case numberOfArgs == 1:
+		fmt.Printf("%s", console.Colorize(cred, "Enter Access Key : "))
+		value, _, _ := reader.ReadLine()
+		accessKey = string(value)
+		fmt.Printf("%s", console.Colorize(cred, "Enter Secret Key : "))
+		bytePassword, _ := terminal.ReadPassword(int(syscall.Stdin))
```

Comment:
Missing proper error handling

Reply:
Also according to [Terminal.ReadPassword](https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword) we are supposed to pass the prompt. In any case we should use `os.File.Fd()` instead of the `syscall` package.